src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,121 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PodcastExternalLinks Should render external links 1`] = `
+.emotion-0 {
+  border-top: 1px #BABABA solid;
```

Comment:
More pixel values here and for border bottom, we should always use a relative value, then this will scale